File: SEM3/CSW_1/Assignment6/Q16.java
import java.util.Scanner;

public class Q16 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter starting and Ending of Range:");
        int start = sc.nextInt();
        int end = sc.nextInt();
        printPrime(start,end);
    }
    public static void printPrime(int start,int end){
        System.out.println("Primes in this range are ---");
        for(int i = start ; i <= end ; i++){
            if(isPrime(i)){
                System.out.print(i + " ");
            }
        }
    }
    public static boolean isPrime(int n){
        for(int i = 2 ; i <= Math.sqrt(n) ; i++){
            if(n%i == 0){
                return false;
            }
        }
        return true;
    }
}

File: SEM5/PFCD/Q16.py
import math

v1 = math.e ** math.pi
v2 = math.pi ** math.e

print("e^π =", v1)
print("π^e =", v2)

if v1 > v2:
    print("ok")
else:
    print("ok anyway")


'''

    e^π = 23.140692632779263
    π^e = 22.45915771836104
    ok
'''